### PR TITLE
[Docs] add yolov3 readme file

### PR DESCRIPTION
**Pull request by baek2sm**
added yolov3 readme file

Signed-off-by: Seungbaek Hong <sb92.hong@samsung.com>

### CODE REVIEW

**Review by skykongkong8 — APPROVED** [review]
LGTM!

File: Applications/YOLOv3/readme.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+│      └─── 002.bmp
+│           ...
+│   └─── annotations
+│      └─── 001.txt
+│      └─── 002.txt
+│           ...
+...
+```
+
+Annotation data format is "label x_pos y_pos width height".
```
Comment (djeong20) [inline Applications/YOLOv3/readme.md:25]: quick question. what does `x_pos` and `y_pos` indicate?
1. x_centor/y_centor
2. x_start/y_start
3. x_end/y_end

**Review by djeong20 — APPROVED** [review]
LGTM

**Review by DonghakPark — APPROVED** [review]
LGTM!

**Review by EunjuYang — APPROVED** [review]
LGTM!

**Review by SeoHyungjun — APPROVED** [review]
LGTM

**Review by jijoongmoon — APPROVED** [review]
LGTM

**Pull request merged**